packages/vizbuilder/src/components/ChartArea/ChartCard.jsx

```diff
@@ -16,6 +17,11 @@ class ChartCard extends React.Component {
 
     config.data = dataset;
 
+    if (type === "lineplot" && relativeStdDev(dataset, config.y) > 1) {
+      config.yConfig.scale = "log";
+      config.yConfig.gridConfig = {"stroke-width": 0};
```

Comment:
If the use defines a specific `gridConfig` (for maybe something other than "stroke-width") and `scale` using the `config` prop, do these overwrite those values? We should only manually set these settings if they have not been defined by the user (or apply the user's config after this)

Reply:
I think that we can fix it in d3plus. I'm going to create an issue.